packages/sources/por-address-list/src/endpoint/utils.ts

```diff
@@ -0,0 +1,25 @@
+import { ethers } from 'ethers'
+
+export const fetchAddressList = async (
+  addressManager: ethers.Contract,
+  confirmations = 0,
+  batchSize = 10,
+): Promise<string[]> => {
+  const blockTag = confirmations ? -confirmations : undefined
```

Comment:
How does this work? Don't you have to fetch the current block height and subtract confirmations? Can you pass in a negative number as the block tag?

Not doubting that it works, just unfamiliar with this

Reply:
Might be safer to do a check of the block number at the start, so the block number between all of these requests doesn't change

Reply:
yeah I think we can pass in a negative number.  I got this info from the docs here https://docs.ethers.io/v5/api/providers/types/#providers-BlockTag

Reply:
Thanks, TIL!

Still think that we should fetch the current block number and subtract, that way we can ensure we are using the same block number for all requests - otherwise we could be calling over two blocks, where the list could have been changed.

Reply:
gotcha ok.  Just updated the PR